app/views/species_list/list_species_lists.html.erb

```diff
@@ -10,7 +10,7 @@
 <%= paginate_block(@pages) do %>
   <% if @objects.any? %>
     <div class="list-group">
-      <% for species_list in @objects.each %>
```

Comment:
Not sure I did this one right. Are these equivalent?

Reply:
After researching a bit, i'm pretty sure these are equivalent.
`for species_list in @objects.each`
`@objects.each do |species_list|`

Reply:
Looks right.